Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxTransformMode.cpp

```diff
@@ -6,50 +6,648 @@
  *
  */
 
-#include "EditorWhiteBoxComponentModeCommon.h"
 #include "EditorWhiteBoxTransformMode.h"
+#include "AzCore/std/smart_ptr/make_shared.h"
```

Comment:
update: This include should be moved down with the `<>` style ones with `AzCore`